alf/trainers/policy_trainer.py

```diff
@@ -273,6 +273,7 @@ def initialize(self):
         if self._evaluate:
             self._eval_env = unwrapped_env
 
+    @gin.configurable('alf.trainers.PolicyTrainer._create_environment')
```

Comment:
Why do you name "PolicyTrainer" instead of just "Trainer"?

Reply:
Should be Trainer. I thought the class name is PolicyTrainer.